classes/module/Module.php

```diff
@@ -758,7 +758,11 @@ public function enable($force_all = false)
             }
         }
 
-        if ($this->getOverrides() != null) {
+        if (count($items) == Shop::getTotalShops()) {
+            $force_all = true;
```

Comment:
Sorry Christian, I think I misspoke.
I totally agree with you, you are pointing at a known issue. For now, we cannot merge this PR because it would introduce one or more regressions.
Core team has planned to discuss about the issue and fix it.